deploy/chart/ingressmonitorcontroller/templates/role.yaml

```diff
@@ -0,0 +1,86 @@
+{{- if and ( .Values.rbac.create ) ( ne .Values.watchNamespaces "") }}
+{{- range .Values.watchNamespaces | split "," }}
+apiVersion: rbac.authorization.k8s.io/v1
+kind: Role
+metadata:
+  name: {{ template "name" $ }}-role
+  namespace: {{ . | quote }}
```

Comment:
Remove `quote` and add `trim`. So, if a user specifies `watchNamespaces` like "test, test-a,test" it would still work perfectly fine.

Reply:
done